packages/runtime/src/dataViews/transport/RelationshipDVO.ts

```diff
@@ -10,6 +10,7 @@ export enum RelationshipDirection {
 export interface RelationshipDVO extends DataViewObject {
     type: "RelationshipDVO";
     status: string;
+    peerDeletionInfo?: PeerDeletionInfoDTO;
```

Comment:
For the deletionInfo of LocalAttributes we flattened it in the DVO, i.e. added two properties `deletionInfo` and `deletionStatus`. If we were to do that here, too, we should probably rename the properties of `peerDeletionInfo` to `peerDeletionStatus` and `peerDeletionDate` to make it clear, that they do not refer to the Relationship itself.